test/controllers/admin/deploy_groups_controller_test.rb

```diff
@@ -0,0 +1,91 @@
+require_relative '../../test_helper'
+
+describe Admin::DeployGroupsController do
+  as_a_deployer do
+    unauthorized :get, :index
+    unauthorized :post, :create
+    unauthorized :delete, :destroy, id: 1
+    unauthorized :post, :update, id: 1
+    unauthorized :get, :new
+  end
+
+  as_a_admin do
+    it 'get index succeeds' do
+      get :index
+      assert_response :success
+      assigns(:deploy_groups).count.must_equal 3
+    end
+
+    unauthorized :post, :create
+    unauthorized :delete, :destroy, id: 1
+    unauthorized :post, :update, id: 1
+    unauthorized :get, :new
+  end
+
+  as_a_super_admin do
+    it 'get :index succeeds' do
+      get :index
+      assert_response :success
+      assert_template :index
+      assert_select('tbody tr').count.must_equal DeployGroup.count
+    end
+
+    it 'get :new succeeds' do
+      get :new
+      assert_response :success
+      assert assigns(:deploy_group)
+    end
+
+    describe '#create' do
+      it 'creates a deploy group' do
+        post :create, deploy_group: {name: 'pod666', environment_id: environments(:staging_env).id}
+        assert_redirected_to admin_deploy_groups_path
+        deploy_groups = DeployGroup.where(name: 'pod666')
+        deploy_groups.count.must_equal 1
```

Comment:
Just out of interest, the first hit on Google for 'minitest assert changed' is this: http://grosser.it/2012/05/10/assert_change-and-assert_no_change-for-testunit-minitest-inspired-by-rspec/